ci-operator/jobs/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main-periodics.yaml

```diff
@@ -1174,14 +1174,6 @@ periodics:
     job-release: "4.10"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.10-e2e-openstack-nfv
-  reporter_config:
```

Comment:
Note for next time, you could have removed the slack notification for the 4.10 job in a separate commit that we could have easily reverted once the job is back to green (after GA IIUC?).

Reply:
good idea. I created the PR already so we won't forget about it.